test/putTransferSpec.js

```diff
@@ -1018,103 +1016,7 @@ describe('PUT /transfers/:id', function () {
         .end()
     })
 
-  /* Subscriptions */
-  it('should trigger subscriptions', function * () {
```

Comment:
Hmm.

Just noticed -- It looks like there is no test for sending notifications with fulfillments attached